src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -299,6 +299,13 @@ module sonic-device_metadata {
                     type string;
                     description "Set source of anchor route";
                 }
+
+                leaf frr_unified_config_restore {
+                    type boolean;
+                    description "When sonic-frr-mgmt-framework module is set to true,this field provides the flexibility to restore
+                        the FRR configurations during startup through frrcfgd.";
+                    default "false";
```

Comment:
default false; ? do we use default with "true"/"false" in other YANG models?

Reply:
with and without double quotes both are commonly used. In this file all the places uses "" so to maintain uniformity retaining the same way.